.golangci.yaml

```diff
@@ -35,8 +35,6 @@ linters-settings:
   errcheck:
     exclude-functions:
       - io.Copy(os.Stdout)
-      - (*github.com/peterstace/simplefeatures/rtree.RTree).RangeSearch
```

Comment:
`errcheck` seems to get a bit confused when generics are introduced... I can longer specify funcs in `exclude-functions` section. Instead, I've opted to just use `_ = ...` whenever I want to ignore an error.